cmake/BLTMacros.cmake

```diff
@@ -219,8 +221,10 @@ macro(blt_add_target_link_flags)
         endif()
         # append new flag
         set(_LINK_FLAGS "${arg_FLAGS} ${_LINK_FLAGS}")
+        # LINK_FLAGS property _must_ be a string
```

Comment:
It might be more helpful if this comment said that it was converting ``_LINK_FLAGS`` from a (cmake) list to a string

Reply:
```suggestion
        # Convert _LINK_FLAGS from a CMake ;-list to a string
```